### PR TITLE
Add autostubbing to `track`.

### LINKED ISSUE #17: `stmocli track` could generate its own filenames
Feature idea for improved ergonomics: `stmocli track` could automatically generate filenames from the query titles instead of requiring a user-specified filename, in the case that the user doesn't really care.

ex. `stmocli track 1012` could read the query title and automatically track as `1012_devtools_dau_mau_er_by_channel.sql`.

### PR DESCRIPTION
**Pull request by tdsmith**
Works like:

```
tsmith-0yhv2t:tsmith stmocli-queries (master)$ stmocli track 1234
Tracking Query ID 1234 in 1234_copy_of_426_android_release_retention_by_experiment_cohort_onboarding_2.sql
```

Closes #17.

### CODE REVIEW

File: stmocli/cli.py [hunk 1]
```diff
@@ -40,6 +42,12 @@ def track(conf, query_id, file_name, redash_api_key):
             urljoin(redash.API_BASE_URL, url_path)
         )
 
+        if not file_name:
```
Comment (harterrt) [inline stmocli/cli.py]: Why include the query id in the name?

Comment (harterrt) [inline stmocli/cli.py]: Can we make this more explicit by using a [prompt](http://click.pocoo.org/5/prompts/) with the default set to your generated filename?

Comment (tdsmith) [reply stmocli/cli.py]: I was using it for a couple reasons:

* something to copy-paste into the URL bar to load something in re:dash (which argues for a `stmocli view` command)
* as part of the devtools-dashboard-refresh task I'm working on, I'm tracking the query IDs of refreshed queries in a spreadsheet, and this made the relationship between the spreadsheet row and the query sql immediately available

So it was the naming pattern that I was using anyway but as long as there's _a_ default I don't think I have strong feelings.

Comment (tdsmith) [reply stmocli/cli.py]: Will do.

**Comment by tdsmith** [comment]
Updated with a prompt.

File: stmocli/cli.py (hunk 1 above)
Comment (harterrt) [reply stmocli/cli.py]: It's not a strong opinion, but prepending the query_id feels like duplicated information to me. Agree we should have a `stmocli view` command.

**Comment by harterrt** [comment]
Friendly ping.

**Pull request merged**